openfecwebapp/templates/advanced.html

```diff
@@ -84,18 +82,13 @@
           </div>
         </div>
         <div class="option__aside">
-          <aside class="card card--primary is-disabled">
-            <span class="card__image card__icon i-committee"><span class="u-visually-hidden">Icon representing a committee</span></span>
-            <div class="card__content">
-              PAC interactive overviews
-            </div>
-          </aside>
-          <aside class="card card--primary card--stacked is-disabled">
-            <span class="card__image card__icon i-committee"><span class="u-visually-hidden">Icon representing a committee</span></span>
-            <div class="card__content">
-              Party committee interactive overviews
-            </div>
-          </aside>
+          <i class="icon-circle--committee is-disabled"><span class="u-visually-hidden">Icon of a committee</span></i>
+            <a href="#" class="is-disabled">PAC data overview</a><br><br>
```

Comment:
`<br><br>` Is probably not the best way to insert space between these items. What's the right way to do it?